packages/amplication-server/src/core/auth/auth.service.ts

```diff
@@ -53,6 +57,8 @@ export class AuthService {
     private readonly passwordService: PasswordService,
     private readonly prismaService: PrismaService,
     private readonly accountService: AccountService,
+    private readonly kafkaProducerService: KafkaProducerService,
```

Comment:
I'd suggest to make the KafkaModule `@Global()` to avoid the need to register it into all needed modules like account.module.

Have a look to this change to know better what I mean: 
https://github.com/amplication/plugins/pull/160/files#diff-d23ed63fdd715eae5265935d400ba5efc5802b528de6f978cbf350a2979decd7R1